content/03.ml-concepts.md

```diff
@@ -1,24 +1,28 @@
 ## Tip 1: Concepts that apply to machine learning also apply to deep learning {#concepts}
-Deep learning is a distinct subfield of machine learning, but it is still a subfield.
-DL has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches in certain contexts, but it is not immune to the many limitations inherent to machine learning.
-Many best practices for machine learning also apply to deep learning.
-Like all computational methods, deep learning should be applied in a systematic manner that is reproducible and rigorously tested.
 
-Those developing deep learning models should select data that are relevant to the problem at hand; non-salient data can hamper performance or lead to spurious conclusions.
-Biases in testing data can also unduly influence measures of model performance, and it may be difficult to directly identify confounders from the model.
-Investigators should consider the extent to which the outcome of interest is likely to be predictable from the input data and begin by throughly inspecting the input data.
-Suppose that there are robust heritability estimates for a phenotype that suggest that the genetic contribution is modest but a deep learning model predicts the phenotype with very high accuracy.
+While deep learning has inarguably surpassed general machine learning for artificial intelligence and predictive modeling applications in terms of public awareness, deep learning still remains a subfield of machine learning.
```

Comment:
The aside about public awareness almost comes across as a little sarcastic/snide, and I don't think it really fits with the rest of the paper's tone.

Reply:
I, for one, jumped straight to deep learning without any knowledge of ML and it was a bit of a train wreck, so I do think that this sentence has value to the complete newcomer. Regardless, if this sentence came off as snide/sarcastic to you (it didn't to me), it is likely that it will to others as well so we should change it.

Reply:
Thanks for the note! Although, I unfortunately have a tendency for sarcasm, I didn't intend to be sarcastic here ... But I agree that we should change it ... Will think about an alternative way to say it